test/e2e/storage/csi_volumes.go

```diff
@@ -58,19 +59,21 @@ var csiTestDrivers = map[string]func(f *framework.Framework, config framework.Vo
 	"[Feature: GCE PD CSI Plugin] gcePD": initCSIgcePD,
 }
 
-var _ = utils.SIGDescribe("CSI Volumes", func() {
+var _ = utils.SIGDescribe("[Serial] CSI Volumes", func() {
```

Comment:
Doesn't this add [Serial] in two distinct places in the test name? I don't believe the whole suite needs to be serial, just the new tests you added

Reply:
We do need the whole suite to be serial - all e2e tests depend on the CRD if either the `CSINodeInfo` or `CSIDriverRegistry` is set, since drivers can't be registered without the CRD.

It's definitely an oversight though to have [Serial] in two places. Will fix that in my subsequent PR